scripts/babel-relay-plugin/src/GraphQLPrinter.js

```diff
@@ -290,6 +290,14 @@ function printFieldsAndFragments(
         fragments.push(printQueryFragment(selection, options));
       } else if (selection.kind === kinds.FIELD) {
         fields.push(selection);
+
+        if (
+          isConnectionType(type) &&
```

Comment:
Yes, let's use `getConnectionMetadata()` (and no need for `!!var`, I've been over-using it to ensure that a value is a boolean - useful with Flow)